mycroft/skills/settings.py

```diff
@@ -140,7 +142,10 @@ def initialize_remote_settings(self):
 
         self._device_identity = self.api.identity.uuid
         self._api_path = "/" + self._device_identity + "/skill"
-        self._user_identity = self.api.get()['user']['uuid']
+        try:
+            self._user_identity = self.api.get()['user']['uuid']
+        except RequestException:
+            return
```

Comment:
Will this change the behavior and stop the polling timer? If so is that good or bad?

Reply:
I don't believe this changes any behavior, as the code above was created using simple "returns" in those cases.  And in my testing observations this was just throwing unhandled exceptions, which obviously didn't do anything smartly either.

Reply:
Actually, you are right!  That _poll_skill_settings() code handles this poorly and needs some love.  This has probably been failing silently for some time.